lib/course_planner/permissions.ex

```diff
@@ -7,9 +7,12 @@ defimpl Canada.Can, for: CoursePlanner.User do
     PageController,
     ScheduleController,
     TaskController,
-    User,
```

Comment:
I prefer leaving lists that are frequently modified with the dangling `,` for better diffs 😄